README.md

```diff
@@ -1,2 +1 @@
-# form-factors
-Foldables and Multi-Screen Explainer
+# Form Factors Explained
```

Comment:
Can you add a link to the preview here?

Reply:
Addressed by c31ffef.

(Once this PR is merged, it'll appear at https://webscreens.github.io/form-factors/)